InformationScripting/src/wrappers/NodeApi.cpp

```diff
@@ -28,15 +28,38 @@
 
 #include "../graph/PropertyMap.h"
 #include "../graph/InformationNode.h"
+#include "../graph/InformationEdge.h"
+#include "../graph/Graph.h"
 
 namespace InformationScripting {
 
 using namespace boost::python;
 
+// The following macro has unused local typdef so we disable the warning for this.
```

Comment:
Hmm, not that pretty but at least it avoids the pragmas here. Could you please try that and see if it works?